Airports/EGHQ/Ownership.txt

```diff
@@ -8,7 +8,7 @@ DEPAPT:EGHQ
 ARRAPT:EGHQ
 
 SECTOR:HQAPP:0:8500
-OWNER:HQR:HQA:LW:LWN:LWC:LWS:L
+OWNER:HQR:HQA:LWL:LWX:LW:LWN:LWC:LWS:L
```

Comment:
```suggestion
OWNER:HQR:HQA:LWH:LWX:LW:LWN:LWC:LWS:L
```
Sector 6 not 9?

Reply:
As above

Reply:
Agreed original is correct for this one. Not for the others!